kolibri/plugins/learn/assets/src/vue/index.vue

```diff
@@ -100,14 +98,8 @@
     padding-left: $left-margin
     padding-right: $right-margin
     padding-bottom: 50px
-
-  .nav
-    position: fixed
-    top: 0
-    left: 0
-    width: $left-margin - $card-gutter * 0.5
-    height: 100%
-    z-index: 2
+    @media screen and (orientation: portrait)
+      padding-left: $left-margin - 20px
```

Comment:
We should avoid hard-coding pixel values. Note that `$card-gutter = 20px` (see _learn.styl_)